src/clj/rems/db/applications.clj

```diff
@@ -138,23 +138,13 @@
   (and (is-applicant? user-id application)
        (= (:state application) "applied")))
 
-(defn translate-catalogue-item [item]
-  (merge item
-         (get-in item [:localizations context/*lang*])))
-
 (defn get-catalogue-items
   "Function that returns localized catalogue-items for the given application items, `ids`. Prefetched localized catalogue items, `localized-items`,
   can be given as a parameter to avoid excessive database calls."
-  ([ids]
-   (mapv translate-catalogue-item
-         (catalogue/get-localized-catalogue-items {:ids ids})))
+  ([ids] (catalogue/get-localized-catalogue-items {:ids ids}))
   ([ids localized-items]
-   (mapv translate-catalogue-item
-         (filter #(some #{(:id %)} ids)
-                 localized-items))))
-
-(defn get-catalogue-item [id]
-  (translate-catalogue-item (catalogue/get-localized-catalogue-item id)))
+   (filter #(some #{(:id %)} ids)
+           localized-items)))
```

Comment:
The 2-parameter version can be removed. It's only used from `get-catalogue-items-by-application-items` which is never used.